weed/shell/replication_health_checker.go

```diff
@@ -0,0 +1,539 @@
+package shell
+
+import (
+	"context"
+	"fmt"
+	"github.com/chrislusf/seaweedfs/weed/glog"
+	"github.com/chrislusf/seaweedfs/weed/operation"
+	"github.com/chrislusf/seaweedfs/weed/pb/master_pb"
+	"github.com/chrislusf/seaweedfs/weed/pb/volume_server_pb"
+	"github.com/chrislusf/seaweedfs/weed/storage"
+	"github.com/chrislusf/seaweedfs/weed/storage/needle"
+	"google.golang.org/grpc"
+	"io"
+	"math/rand"
+	"sort"
+	"strings"
+	"sync"
+)
+
+func init() {
+	//commands = append(commands, &commandReplicationHealthChecker{})
+	//commands = append(commands, &commandReplicationHealthRepair{})
+}
+
+type commandReplicationHealthChecker struct {
+	args       []string
+	commandEnv *commandEnv
+	writer     io.Writer
+	checker    *ReplicationHealthChecker
+}
+
+func (c *commandReplicationHealthChecker) Name() string {
+	return "volume.check.replication"
+}
+
+func (c *commandReplicationHealthChecker) Help() string {
+	return `
```

Comment:
This can be a critical command. Need help message.

Need to describe what volumes are considered unhealthy, and what are considered healthy, and what operations will be taken to fix the unhealthy volumes.